src/TLuaInterpreter.h

```diff
@@ -196,7 +195,6 @@ class TLuaInterpreter : public QThread
     static int setCustomEnvColor( lua_State * );
     static int roomLocked( lua_State * );
     static int setAreaName( lua_State * );
-    static int setRoomColor( lua_State * );
```

Comment:
This has been supplanted I think by "setRoomEnv" but I.R.E. Xml Maps do some really funky things to confuse the hell out of things by mapping {in a __non__-User visible/editable way} what, I think, they call "envId"s to "colors" however when we use `setRoomEnv` we are actually setting a colour, though we call it `(int) TRoom::environment` - just see `(QColor) T2DMap::getColor( int id )` to see the confusing way this is set up! :roll_eyes:

Anyhow this is currently redundant as you have spotted... :stuck_out_tongue_winking_eye: